packages/stark-ui/src/modules/breadcrumb/components/breadcrumb.component.scss

```diff
@@ -0,0 +1,52 @@
+/* ============================================================================== */
+/*  S t a r k   B r e a d c r u m b                                               */
+/* ============================================================================== */
+/* stark: src/ui/components/breadcrumb/breadcrumb.component.pcss */
+//.stark-breadcrumb {
+//  display: none;
```

Comment:
I see in the old stark, the display was "none" by default and changed if it was desktop. Why changing this?
Does it mean we want to show this element on mobile ?

Reply:
Sorry my bad, I've forgotten to uncomment after I've done all my tests